pkg/cmd/attestation/verification/tuf.go

```diff
@@ -0,0 +1,55 @@
+package verification
+
+import (
+	"embed"
+	"fmt"
+	"os"
+
+	"github.com/sigstore/sigstore-go/pkg/tuf"
+)
+
+//go:embed embed
+var embeddedRepos embed.FS
+
+const GitHubTUFMirror = "https://tuf-repo.github.com"
+
+// readEmbeddedRoot reads the embedded trust anchor for the given URL
+func readEmbeddedRoot(url string) ([]byte, error) {
+	// the embed file system always uses forward slashes, even on Windows
+	p := fmt.Sprintf("embed/%s/root.json", tuf.URLToPath(url))
```

Comment:
As mentioned above, all path generation should really be using `path.Join`

Reply:
So this piece of code has to use `fmt.Sprintf` instead of `path.Join` because it is building a path to a file that is part of the embedded file system, which [always uses forward slashes](https://pkg.go.dev/embed#hdr-Directives), even on Windows. The unit tests testing this piece of code would fail when run on the Windows platform. I can add some more info to the comment and the link to the embed documentation.